<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoiMatKhau</class>
 <widget class="QMainWindow" name="DoiMatKhau">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>606</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>368</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Đổi Mật Khẩu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form
{
	background-image: url(:/background/BACKGROUND/bg1.jpg);
}</string>
  </property>
  <widget class="QWidget" name="Form">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>80</y>
      <width>301</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color: #222;
	border-radius: 20px
}

QPushButton
{
	background: red;
	border-radius: 10px;
}

QLineEdit
{
	background: transparent;
	border:none;
	color:#717072;
	border-bottom: 1px solid #717072;
}

QLabel
{
	color: white;
	font: 900 18pt &quot;Segoe UI&quot;;
}

QGroupBox
{
	border: 2px solid rgb(255, 255, 127);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>281</width>
       <height>401</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="ID_LABEL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>8</y>
        <width>41</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="NP_LABEL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New pass:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ID_LINE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter Your ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NP_LINE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter New Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DMK_BUTTON">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CHANGE PASSWORD</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="CF_LABEL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Confirm:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CF_LINE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Confirm Your Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EXIT_BUTTON">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>346</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 85, 127);</string>
      </property>
      <property name="text">
       <string>GO BACK</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>54</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>CHANGE PASSWORD</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>31</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
	background-color: rgb(255, 151, 250);
	
	
	border-radius: 50px;;
}
</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="ing.qrc">
      <normaloff>:/icon/ICON/reset-password.png</normaloff>:/icon/ICON/reset-password.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ing.qrc"/>
 </resources>
 <connections/>
</ui>
